Community/Tdarr_Plugin_bsh1_Boosh_FFMPEG_QSV_HEVC.js

```diff
@@ -0,0 +1,400 @@
+// All credit for original plugin logic goes to Migz.
+// This Plugin is essentially just his NVENC/CPU plugin modified to work with QSV & with extra hevc logic.
+// Extra logic is mainly to control encoder quality/speed & to allow HEVC files to be reprocessed to reduce file size
+// NOTE - This does not use VAAPI, it is QSV only. So newer intel igpus only. 8th+ gen should work.
```

Comment:
Is the driver support up to snuff now for the containers? Plus, how would people use this on Linux bare metal?

Reply:
Driver support is now fixed in containers. - https://github.com/HaveAGitGat/Tdarr/issues/452
FFMPEG in the container is using jellyfins version now which has QSV support packaged in ready to use. So container users just need a valid plugin and the right hardware to leverage QSV.
As for bare metal users, assuming Tdarr packages jellyfin ffmpeg there as well then they should just need to install up to date drivers and that should work. I can't verify that though.